File: core/CreateTournament/Models/Team.cs
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace CreateTournament.Models
{
    public class Team: DeleteableEntity
    {
        public string Name { get; set; }
        public string? ImageTeam { get; set; }
        public int? Point { get; set; }
        public Collection<Player> Players { get; set; }
        public int TournamentId { get; set; }
        [ForeignKey("TournamentId")]
        public Tournament Tournament { get; set; }
    }
}

File: client/src/app/core/models/classes/Team.ts
import { ITeam } from "../interfaces/ITeam";

export class Team implements ITeam {
	id: number;
	name: string;
	imageTeam?: string | undefined;
	point?: number | undefined;
	tournamentId: number;

	constructor() {
		this.id = 0;
		this.name = "Team";
		this.imageTeam = "";
		this.point = 0;
		this.tournamentId = 0;
	}
}
